project/src/main/webapp/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This file is junk now right?

Reply:
Yes you are right! f90372ec688566aba6c650a6f740a1dab6477c1a